themes/openy_themes/openy_lily/templates/node/node--program--full.html.twig

```diff
@@ -83,6 +83,15 @@
 
 {# Program header #}
 <article{{ attributes.addClass('program-header') }}>
+  {% if content.field_header_content is not empty %}
+  <div class="container">
+    <div class="row">
```

Comment:
These wrappers won't allow the content to take the whole page width.

Reply:
Fixed